### DISCUSSION TITLE
Is it possible to use `moon` in a polyglot monorepo?

### DISCUSSION THREAD

**Discussion by ydennisy** [moonrepo/moon#1638] [Q&A] [answered]
We have a monorepo of TS, Python & Go apps, services & libs.

From looking at the examples right now only Rust & TS is supported is that correct? Also I have not seen any polyglot examples, is that supported?

**Comment by milesj** [comment, marked as answer]
@ydennisy You can run anything in moon, since it's just a task runner, and it will run whatever is available on `PATH`.

Supported languages (https://moonrepo.dev/docs#supported-languages) just provide deeper integration, like lockfile inspection and automatic dependency installation. That kind of stuff won't exist for python/go, but you can still run python/go commands.

**Reply by ydennisy** [reply]
Thanks @milesj !

Maybe it is just me - but I have read over multiple examples, the site and docs on a few sittings and I am still not sure what the point of moon is and where the project will be heading.

When you say it is just a task manager, that is ok - but the site seems to suggest it is a monorepo management tool, which is what I was after but to me it is not clear why I would use this.

All the best!

**Reply by milesj** [reply]
@ydennisy It's a monorepo tool because it offers more than a task runner, like code/project ownership, project boundaries, docker integration, CI support, webhooks, and more. Things you'll need once a monorepo grows in size.

And also back to the original question, it's polyglot so that multiple languages can all be in the same project/task graph, so that they can depend on each other correctly.
